src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -51,3 +53,29 @@
    (-> wallet
        (get-in [:accounts current-viewing-account-address])
        (assoc :balance (get balances current-viewing-account-address)))))
+
+(defn- calc-token-value
+  [{:keys [symbol market-values-per-currency] :as item}]
+  (let [fiat-value                        (utils/total-token-value-in-all-chains item)
+        market-values                     (:usd market-values-per-currency)
+        {:keys [price change-pct-24hour]} market-values
+        fiat-change                       (utils/calculate-fiat-change fiat-value change-pct-24hour)]
+    {:token               (keyword (string/lower-case symbol))
+     :state               :default
+     :status              (cond
+                            (pos? change-pct-24hour) :positive
+                            (neg? change-pct-24hour) :negative
+                            :else                    :empty)
+     :customization-color :blue
+     :values              {:crypto-value      (.toFixed (* fiat-value price) 2)
+                           :fiat-value        (utils/prettify-balance fiat-value)
+                           :percentage-change (.toFixed change-pct-24hour 2)
+                           :fiat-change       (utils/prettify-balance fiat-change)}}))
+
+(rf/reg-sub
+ :wallet/account-token-values
+ :<- [:wallet]
+ :<- [:wallet/accounts]
+ (fn [[{:keys [current-viewing-account-address]} accounts]]
```

Comment:
@OmarBasem we already have a sub to `:wallet/current viewing-account`, it also has the key `:token`.
So this sub could only depend on it to get the same info 👍